narwhals/_dask/expr.py

```diff
@@ -438,6 +438,34 @@ def abs(self) -> Self:
             returns_scalar=False,
         )
 
+    def all(
+        self,
+        *,
+        ignore_nulls: bool = True,
```

Comment:
we don't pass down any arguments in Narwhals, I think you can just remove it from here and set the value which matches Polars' default in the function body